kasa/tests/test_discovery.py

```diff
@@ -68,7 +75,9 @@ def mock_discover(self):
     mocker.patch.object(_DiscoverProtocol, "do_discover", mock_discover)
     mocker.patch("kasa.TPLinkSmartHomeProtocol.query", return_value=discovery_data)
 
-    x = await Discover.discover_single(host, port=custom_port)
+    x = await Discover.discover_single(
+        host, port=custom_port, credentials=Credentials("user", "pass")
```

Comment:
```suggestion
        host,
        port=custom_port,
        credentials=Credentials("user", "pass")
```

Reply:
I'm not sure if this is applicable for a test, but the default username/password for the EC70 appears to be admin/admin.  But the password is base64 encoded.  Which I believe works out to `YWRtaW4=`.

Reply:
This can be tested with 
```
curl -vv -k -u admin:YWRtaW4= "https://192.168.0.10:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd"
```

or 
```
curl -vv -k -u admin:$(echo -n "admin" | base64) "https://192.168.0.10:19443/https/stream/mixed?video=h264&audio=g711&resolution=hd"
```